docs/guide/tutorial-3-appearance-15-minutes.md

```diff
@@ -8,19 +8,17 @@ The result at the end of this tutorial is shown in [this figure](#simulation-aft
 
 ### New Simulation
 
-> **Hands on**: From the results of the previous tutorial, create a new simulation called "appearance.wbt" by using the `File / Save World As...` menu.
+> **Hands on #1**: From the results of the previous tutorial, create a new simulation called `appearance.wbt` by using the `File / Save World As...` menu.
 
 ### Lights
 
-> **Theory**: The lighting of a world is determined by light nodes.
+The lighting of a world is determined by light nodes.
 There are three types of light nodes: the DirectionalLight, the [PointLight](../reference/pointlight.md) and the [SpotLight](../reference/spotlight.md).
 A [DirectionalLight](../reference/directionallight.md) simulates a light which is infinitely far (ex: the sun), a [PointLight](../reference/pointlight.md) simulates light emitted from a single point (ex: a light bulb), and a [SpotLight](../reference/spotlight.md) simulates a conical light (ex: a flashlight).
 Each type of light node can cast shadows.
 You can find their complete documentation in the [Reference Manual](../reference/nodes-and-api-functions.md).
 
-<!-- -->
-
-> **Note**: Lights are costly in term of performance.
+Lights are costly in term of performance.
 Minimizing the number of lights increases the rendering speed.
 A maximum of 8 lights is allowed if the shaders are disabled in the Webots preferences.
```

Comment:
It is not possible anymore to disable the shaders in the current version of Webots.
If I am not wrong the maximum number of lights is now 256 per type in any case, but we should probably omit this in the tutorial.